src/App.js

```diff
@@ -0,0 +1,23 @@
+import './App.css';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import NavBar from './components/navBar';
+import HomePage from './components/homePage';
+import Categories from './components/categories';
+import Footer from './components/footer';
+
+function App() {
+  return (
+    <div className="App">
+      <NavBar />
+      <BrowserRouter>
+        <Routes>
+          <Route path="/" element={<HomePage />} />
+          <Route path="/categories" element={<Categories />} />
```

Comment:
- [ ] I am unable to navigate to the categories component. As can see I have clicked the categories navbar link yet I am still on the Homepage. Kindly fix this.

  ![image](https://user-images.githubusercontent.com/54848363/207798004-e47fbb68-6750-40ef-b739-9e46db48c71b.png)